src/main/java/com/openlattice/chronicle/services/delete/DataDeletionService.java

```diff
@@ -254,15 +281,15 @@ private void legacyDeleteStudyData(
                     deleteType );
 
             // delete participants
-            userDataApi.deleteEntities( participantsESID, participantsToDelete, deleteType, false );
+            chronicleDataApi.deleteEntities( participantsESID, participantsToDelete, deleteType, false );
```

Comment:
No change required. `chronicleDataApi` and `userDataApi` are confusing. It should be either prod or integration.